extensions/mssql/src/objectManagement/ui/databaseDialog.ts

```diff
@@ -705,6 +705,12 @@ export class DatabaseDialog extends ObjectManagementDialogBase<Database, Databas
 				isEnabled = false;
 			}
 		}
+		else if (table === this.filestreamFilegroupsTable && this.filestreamFilegroupsTable.selectedRows !== undefined && this.filestreamFilegroupsTable.selectedRows.length === 1) {
+			// Disable remove button when server filestream access level is disabled.
+			if (this.viewInfo.serverFilestreamAccessLevel === null || this.viewInfo.serverFilestreamAccessLevel === 0) {
```

Comment:
reason for using '0', STS FileStreamEffectiveLevel enum has the Disabled(0) value
![image](https://github.com/microsoft/azuredatastudio/assets/74571829/7e7ec9f6-e818-46e6-8c78-81a5825c8d92)